src/Subscription.php

```diff
@@ -82,6 +82,19 @@ public function active()
         return is_null($this->ends_at) || $this->onGracePeriod();
     }
 
+    /**
+     * Filter query by active.
+     *
+     * @param  Illuminate\Database\Eloquent\Builder  $query
```

Comment:
Please prepend all the namespaces with a `\`

Reply:
🤦‍♂️ sorry, my bad. That's done 👍